recipe-client-addon/lib/NormandyApi.jsm

```diff
@@ -62,7 +62,10 @@ this.NormandyApi = {
   },
 
   async getApiUrl(name) {
-    const apiBase = prefs.getCharPref("api_url");
+    let apiBase = prefs.getCharPref("api_url");
```

Comment:
I think a perfectly legit question to ask is: why does this not have a guaranteed format? Can we make sure that this pref has a trailing slash?

[This search](https://github.com/mozilla/normandy/search?utf8=%E2%9C%93&q=api_url&type=) suggests a bunch of places where we might fix the string instead of doing this.